roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -3,12 +3,13 @@ kafka_broker_jolokia_java_arg_buildout: "-javaagent:{{jolokia_jar_path}}=port={{
 kafka_broker_jmxexporter_java_arg_buildout: "-javaagent:{{jmxexporter_jar_path}}={{kafka_broker_jmxexporter_port}}:{{kafka_broker_jmxexporter_config_path}}"
 kafka_broker_ssl_java_arg_buildout: "-Djavax.net.ssl.keyStore={{kafka_broker_keystore_path}} -Djavax.net.ssl.trustStore={{kafka_broker_truststore_path}} -Djavax.net.ssl.keyStorePassword={{kafka_broker_keystore_storepass}} -Djavax.net.ssl.trustStorePassword={{kafka_broker_truststore_storepass}}"
 kafka_broker_log4j_java_arg_buildout: "-Dlog4j.configuration=file:{{kafka_broker.log4j_file}}"
-
+kafka_broker_heap_opts: "-Xmx1g"
 kafka_broker_custom_log4j: true
 
 kafka_broker_custom_java_args: ""
 kafka_broker_java_args:
   - "{{ kafka_broker_jaas_java_arg_buildout if 'GSSAPI' in kafka_broker_sasl_enabled_mechanisms else '' }}"
+  - "{{ kafka_broker_jaas_java_arg_buildout if zookeeper_sasl_digestmd5_enabled else '' }}"
```

Comment:
should there be an or here? ie
```
kafka_broker_java_args:
  - "{{ kafka_broker_jaas_java_arg_buildout if 'GSSAPI' in kafka_broker_sasl_enabled_mechanisms or zookeeper_sasl_digestmd5_enabled|bool else '' }}"
```

Reply:
Well, as GSSAPI and DigestMD5 can not be used at the same time - both approaches are possible. 
But yours looks clearer, I agree